department-of-veterans-affairs/va.gov-cms
# VACMS-5399: Hide extraneous menu fields on node form.

## Description
See #5399

## Testing done
Visual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/119815777-e8dda880-beb9-11eb-9adf-def8773876f8.png)

## QA steps
As an administrator, go to `/node/1139/edit` and visually verify that menu description and weight fields are not visible.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
